From: Jakub Jirutka <jakub@jirutka.cz>
Date: Thu, 12 Oct 2023 01:05:47 +0200
Subject: [PATCH] Fix unknown type name 'int64_t'

    guestfs-utils.h:62:32: error: unknown type name 'int64_t'
       62 | extern int guestfs_int_is_reg (int64_t mode);
          |                                ^~~~~~~
    guestfs-utils.h:40:1: note: 'int64_t' is defined in header '<stdint.h>'; did you forget to '#include <stdint.h>'?
       39 | #include "cleanups.h"
      +++ |+#include <stdint.h>
       40 | #include "guestfs-stringlists-utils.h"

--- a/common/utils/guestfs-utils.h
+++ b/common/utils/guestfs-utils.h
@@ -34,6 +34,7 @@
 
 #include <stdio.h>
 #include <stdbool.h>
+#include <stdint.h>
 
 #include "guestfs-internal-all.h"
 #include "cleanups.h"
